src/firewall.lib.php

```diff
@@ -665,10 +665,14 @@ public static function deblocklistAjax()
      * @param  array|bool $api_key The firewall API key.
      * @return string|bool         Message explaining the result of the operation.
      */
-    public static function clearCache($api_key = false)
+    public static function clearCache($api_key = false, $path=null)
```

Comment:
should we maybe adjust the spacing on this variable declaration to be consistent?